src/content/docs/logs/forward-logs/fluent-bit-plugin-log-forwarding.mdx

```diff
@@ -54,6 +54,23 @@ To install the Fluent Bit plugin:
   If you'd rather not compile the plugin yourself, you can download pre-compiled versions from our [GitHub repository's releases page](https://github.com/newrelic/newrelic-fluent-bit-output/releases).
 </Callout>
 
+## Upgrade the Fluent Bit plugin [#upgrade-plugin]
+
+Before you upgrade your Fluent Bit plugin, run the following NRQL query to find the current versions of the plugin being used in your system:
+
+```sql
+    FROM
+    K8sContainerSample select latest(containerImage) where podName like '%newrelic-logging%' facet clusterName
+```
+
+<Callout variant="caution">
+    Fluent Bit versions 2.0.7 through 3.0.3 are affected by a [security vulnerability (CVE-2024-4323)](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-4323). If you're using one of these versions, upgrade to a different version. For more info on this, see our [security bulletin NR24-01 - Fluent Bit](https://docs.newrelic.com/docs/security/new-relic-security/security-bulletins/security-bulletin-nr24-01/).
```

Comment:
@paperclypse - the query returns the version of the NR FB output plugin not the FB version. the users need to check if output plugin version they are using is affected: 
[newrelic-fluent-bit-output-1.16.0](https://github.com/newrelic/newrelic-fluent-bit-output/releases/tag/v1.16.0) to [newrelic-fluent-bit-output-1.19.2](https://github.com/newrelic/newrelic-fluent-bit-output/releases/tag/v1.19.2)

Reply:
@Tcohn1 I addressed this in my latest commit